convene-web/app/controllers/invitations_controller.rb

```diff
@@ -3,8 +3,12 @@ class InvitationsController < ApplicationController
   # TODO: fix create action to create invitation data so spec test passes
   def create
     @space_invitation = current_space.invitations.new(invitation_params)
-    @space_invitation.save!
-    redirect_to [current_space, @space_invitation]
+    if @space_invitation.save!
+      SpaceInvitationMailer.space_invitation_email(@person).deliver_later
```

Comment:
Could you clarify where you expect `@person`  to be set and/or where it came from? I'm looking through the "flow" of the code; and it seems like it's going to be a `nil` value.

Also, clarifying point: `save!` will return either `true` or raise an exception; so we will never reach the `else` branch. A more "railsy" way would be to use `save` without the bang; and then render the `new` action so any validation errors are presented back to the user to be corrected.